extensions/jms/src/main/java/org/mule/extensions/jms/internal/connection/provider/activemq/ActiveMQConnectionFactoryProvider.java

```diff
@@ -77,7 +76,7 @@ ConnectionFactory createDefaultConnectionFactory() throws ActiveMQConnectionExce
       String message = format("Failed to create a default Connection Factory for ActiveMQ using the [%s] implementation: ",
                               getFactoryClass());
       LOGGER.error(message, e);
-      throw new ActiveMQConnectionException(createStaticMessage(message), e);
+      throw new ActiveMQException(message, e);
```

Comment:
Why? each ConnectionProvider, except the generic one, is bounded to a broker and is prone to having custom errors. 
In this case, we are not being able to instantiate an ActiveMQ factory, it is a connection error that can happen in no other provider.